Daiataxis template for documentation is ready. (#33)
FAIRmat-NFDI/pynxtools-stm: docs/how-to-guides/how-to-interact-with-reader.md

> # Use of STM Reader
> ## How To Install STM Reader
> Installing STM reader is as simple as installing python package from PyPI. We recommend to install the reader in a python virtual environment.

Reviewer: ```suggestion
# How to Use the STM Reader
## How To Install the STM Reader
The STM reader can be installed as a python package from PyPI. We recommend to install the reader in a python virtual environment.
```

Author: Resolved!

### Installation of STM reader in python virtual envrironment
Run the followoing command step-by-step on your linux terminal

```bash
<user>$ vitualenv --python=3.10 .pyenv
<user>$ source activate .pyenv/bin/activate
(.pyenv) <user>$ pip install pynxtools-stm
```

That's it! You can run the `STM` reader from command line.

### Installation of STM reader in development mode in python virtual environment

> Install `STM` reader to develop according to you own expectation

Reviewer: ```suggestion
If you want to make changes to the `STM` reader, you can install in in to development mode:```

Author: Resolved!
